breathecode/feedback/models.py

```diff
@@ -99,7 +102,9 @@ def __init__(self, *args, **kwargs):
     highest = models.CharField(max_length=50, default="very likely")
     lang = models.CharField(max_length=3, blank=True, default="en")
 
-    platform_and_content = models.BooleanField(default=False)
+    aditional_question = models.CharField(
```

Comment:
There is a typo https://www.oxfordlearnersdictionaries.com/us/definition/english/additional?q=additional

Reply:
`additional_question` isn't descriptive enough